lib/oli_web/controllers/pow.ex

```diff
@@ -11,23 +11,37 @@ defmodule OliWeb.PowController do
   @ttl :timer.minutes(24 * 60)
   @cache_config {PowResetPassword.Store.ResetTokenCache, ttl: @ttl}
 
-  def create_user_password_reset_link(%{"id" => id}) do
-    %{email: email} = _user = Repo.get(User, id)
+  @doc """
+  Create a password reset link for a user or author that will expire in
+  the amount of time defined in @ttl (currently 24 hours).
+
+  id: the id of the user or author
+  account_type: :user or :author
+  """
```

Comment:
Thanks for adding the docs. Including a spec would also be beneficial for a quick overview.